backend/app/main.py

```diff
@@ -87,14 +124,10 @@ async def get_listing(
     except Exception as e:
         return Field500ErrorResponse(error="Internal Server Error. Please try again later.")
 
-
+          
 @app.get('/listings', 
     response_model=ListingsGetAllResponse, responses={'500': {'model': Field500ErrorResponse}},)
 async def get_listings() -> Union[ListingsGetAllResponse, Field500ErrorResponse]:
-    """
-    Retrieve all listings
-    """
-    try:
         # Fetch all listings from MongoDB
```

Comment:
I think this was a merge mistake: you accidentally deleted the `try` and docstring